messaging-activemq/src/main/resources/schema/wildfly-messaging-activemq_13_0.xsd

```diff
@@ -0,0 +1,1043 @@
+                    <xs:attribute name="transformer-class-name" type="xs:string" use="optional" />
+                    <xs:attribute name="exclusive" type="xs:boolean" use="optional" />
+                </xs:complexType>
+            </xs:element>
+
+            <xs:element name="bridge" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="credential-reference" type="credential-reference:credentialReferenceType" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation>
+                                    Credential to be used by the configuration.
+                                </xs:documentation>
+                            </xs:annotation>
+                        </xs:element>
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:string" use="required" />
+                    <xs:attribute name="queue-name" type="xs:string" use="required" />
+
+                    <xs:attribute name="forwarding-address" type="xs:string" use="optional" />
+                    <xs:attribute name="ha" type="xs:boolean"  use="optional" />
+                    <xs:attribute name="filter" type="xs:string"  use="optional" />
+                    <xs:attribute name="transformer-class-name" type="xs:string"  use="optional" />
+                    <xs:attribute name="min-large-message-size" type="xs:int" use="optional" />
+                    <xs:attribute name="check-period" type="xs:long"  use="optional" />
+                    <xs:attribute name="connection-ttl" type="xs:long"  use="optional" />
+                    <xs:attribute name="retry-interval" type="xs:long"  use="optional" />
+                    <xs:attribute name="retry-interval-multiplier" type="xs:double"  use="optional" />
+                    <xs:attribute name="max-retry-interval" type="xs:long"  use="optional" />
+                    <xs:attribute name="reconnect-attempts" type="xs:int"  use="optional" />
+                    <xs:attribute name="reconnect-attempts-on-same-node" type="xs:int"  use="optional" />
+                    <xs:attribute name="use-duplicate-detection" type="xs:boolean"  use="optional" />
+                    <xs:attribute name="confirmation-window-size" type="xs:int"  use="optional" />
+                    <xs:attribute name="producer-window-size" type="xs:int" use="optional" />
+                    <xs:attribute name="user" type="xs:string"  use="optional" />
+                    <xs:attribute name="password" type="xs:string"  use="optional" />
+                    <xs:attribute name="static-connectors" type="stringList"  use="optional" />
+                    <xs:attribute name="discovery-group" type="xs:string"  use="optional" />
+                    <xs:attribute name="call-timeout" type="xs:long"  use="optional" />
```

Comment:
maybe you should set the default value there

Reply:
Done.